AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -0,0 +1,75 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+/// A component that presents a QR code.
+internal final class QRCodeComponent: Component, Localizable {
```

Comment:
`QRCodeComponent ` is an `ActionComponent`?

Reply:
yes, you are correct 👍